packages/components/src/navigation/NavTab.tsx

```diff
@@ -67,17 +67,21 @@ const NavTab = memo(
               data-testid={`btn-nav-tab-${title}`}
               role="tab"
               tabIndex={0}
-              onAuxClick={event => {
-                // e.button equaling 1 corresponds to middle mouse button being clicked
-                if (isClosable && event.button === 1) {
+              onAuxClick={e => {
```

Comment:
I think this needs to go in the `onMouseUp` prop to work in Safari right?

Reply:
Spoke to Matt, and the issue with the onMouseUp is that there is no track of if you started the click somewhere else, tried this on Safari and seems to be working.